template/public/components/main/main.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  EuiPage,
+  EuiPageHeader,
+  EuiTitle,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentBody,
+  EuiText
+} from "@elastic/eui";
+
+export const Main = ({ title, currentTime }) => {
+  return (
+    <EuiPage>
+      <EuiPageHeader>
+        <EuiTitle size="l">
+          <h1>{title} Hello World!</h1>
+        </EuiTitle>
+      </EuiPageHeader>
+      <EuiPageBody>
```

Comment:
I've gotten in the habit of adding a newline between large adjacent elements in JSX, which helps me scan and form an idea of the resulting DOM structure. Applied here, this would add newlines to line 21 and 28. Something to consider.